config/sync/core.entity_form_display.node.article.default.yml

```diff
@@ -0,0 +1,125 @@
+uuid: 94f4f567-c039-46c9-91f9-1e7b8e34d508
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.article.field_a_body
+    - field.field.node.article.field_a_content_author
+    - field.field.node.article.field_a_headline
```

Comment:
Delete this field you can reuse the title field and rename it to headline in the content type settings

Reply:
done